playbook/app/pb_kits/playbook/pb_multiple_users/_multiple_users.jsx

```diff
@@ -26,14 +27,15 @@ const MultipleUsers = (props: MultipleUsersProps) => {
     id,
     maxDisplayedUsers = 4,
     reverse = false,
+    size = 'xs',
     users,
   } = props
 
   const displayCount =
     users.length > maxDisplayedUsers ? maxDisplayedUsers - 1 : users.length
   const usersToDisplay = users.slice(0, displayCount)
-
   const reverseClass = reverse === true ? 'reverse' : ''
+  const avatarSize = () => size === 'xxs' ? '_xxs' : ''
```

Comment:
@derricklannaman I think instead of a function you could follow the `reverseClass` pattern above. Then include the 'avatarSize` in the `buildCss` function like `reverseClass`.  The empty still would become `null` so extra characters aren't included in the classname on line 65.

Also, I could be reading this wrong but is this line eliminating the default `xs` value?

For example, Prop `size = "xs"` then that is evaluated `"xs" === "xss"` it false then becomes "".

It might be acting differently since it's assigned `const` and I might be misunderstanding functionality. It just stood out to me as something worth mentioning. Disregard if necessary.

CC: @kevinschlabach

Reply:
@coleerikson On Multiuser, the default is always `xs`, however we check to see if there is a size prop passed in and if there is, then we change the classname from `multiple_users_badge` to `multiple_users_badge_xxs`. That will adjust the width and height of the round badge. I can look into using the `buildCss` function. Thanks!